### ISSUE TITLE
GTK app: file list ignores GNOME text scaling factor

### ISSUE THREAD

**Issue by bravely-beep** [transmission/transmission#7093]
### What is the issue?

When opening a new torrent, or viewing the properties window for an existing torrent, the file list is always rendered at a fixed resolution. This ignores the `org.gnome.desktop.interface text-scaling-factor` dconf setting. This is an accessibility concern: the "Large text" accessibility setting in the GNOME settings app does not affect the Transmission file list.

Attached screenshot shows the torrent settings window with GNOME `text-scaling-factor` set to `1.35`, increased from the deafault `1.0`. All of the text in the window has increased in size to match the scaling setting, except for the file list.

![image](https://github.com/user-attachments/assets/8324493b-4539-49cd-853d-170d7989bdea)

(I'm running the `transmission-gtk` pacman package on Arch Linux, using the GNOME Shell v46.4 desktop environment)

### Which application of Transmission?

GTK+ app on Linux, BSD, etc.

### Which version of Transmission?

4.0.6 (38c164933e)

**Comment by mikedld** [comment]
"Have" and "Priority" columns seem to be fine. Font size for "Name" and "Size" columns is being reduced to 80% (on purpose), which I guess doesn't work all that nice. Maybe we shouldn't be playing with the font size there at all, otherwise we'd want to find a better way that takes scaling factor into account, including when it changes dynamically while the program is running.
